src/azure-cli/azure/cli/command_modules/acs/_help.py

```diff
@@ -2072,7 +2072,7 @@
         - name: --roles
           type: string
           short-summary: Specify the comma-separated roles.
-        - name: --source-resource-id -r
+        - name: --source-resource-id
```

Comment:
What are the considerations for removing this option?

As this is a breaking change, need to wait until the next breaking change window. Before that, you could mark it as deprecated.

Reply:
After discussed we decided to remove the alias '-r'. 

As TA command hasn't been released ever, we can make this change before this version's release

Reply:
Agree, the option to be removed was introduced in #28171 on 01/12 and has not been released